rainier-test/src/test/scala/com/stripe/rainier/compute/ComputeTest.scala

```diff
@@ -0,0 +1,16 @@
+package com.stripe.rainier.compute
+
+import org.scalatest._
+
+trait ComputeTest extends FunSuite {
+  def assertWithinEpsilon(x: Double, y: Double, clue: String): Unit = {
```

Comment:
making `clue: => String` can make a lot of tests more efficient because you can skip evaluating clue in some cases.